CakePHP/ruleset.xml

```diff
@@ -147,6 +147,9 @@
             <property name="allowFallbackGlobalConstants" type="boolean" value="true"/>
         </properties>
     </rule>
+    <rule ref="SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly.ReferenceViaFullyQualifiedName">
```

Comment:
But this will affect all FCQN, not just attributes isn't it? We do want to ensure `use` statement are used instead of FQCN for regular classes.

Reply:
OK, Then I guess we need a different solution.
We just have a different FQCN check on top that only affects use statements.

But fair enough.